mkdocs.yml

```diff
@@ -44,6 +44,8 @@ nav:
       - Locating and Ingesting Image Data: getting-started/using-isis-first-steps/locating-and-ingesting-image-data.md
       - Adding SPICE:                      getting-started/using-isis-first-steps/adding-spice.md
       - Exporting ISIS Data:               getting-started/using-isis-first-steps/exporting-isis-data.md
+    - Software Management:
+      - Contributing with Git:             getting-started/software-management/contributing-with-git.md
```

Comment:
There is already a software management group, this causes it to override that other group and clobber them. I also dont think this belongs under software management. Probably "contributing" and put it first on the list. Maybe also put the issue page under the same category.